src/tests/ftest/util/test_utils_pool.py

```diff
@@ -254,6 +256,30 @@ def destroy(self, force=1):
 
         return status
 
+    @fail_on(DaosApiError)
```

Comment:
Since this function is only using the `DmgCommand`, and not the `DaosPool` (API), to issue commands, this fail_on should be for a  `CommandFailure`